### PR TITLE
backoffice: disable email verification

### PR DESCRIPTION
**Pull request by DonHaul**
*ref: cern-sis/issues-inspire/issues/604

### CODE REVIEW

**Review by drjova — APPROVED** [review]
Thank you, only in the commit message a space between `*` and `r`

File: backoffice/config/settings/base.py [hunk 1]
```diff
@@ -332,7 +332,7 @@
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
 ACCOUNT_USER_MODEL_USERNAME_FIELD = None
 # https://django-allauth.readthedocs.io/en/latest/configuration.html
-ACCOUNT_EMAIL_VERIFICATION = False
+ACCOUNT_EMAIL_VERIFICATION = "none"
```
Comment (drjova) [inline backoffice/config/settings/base.py:335]: Shouldn't this be `None`?

**Review by PascalEgn — APPROVED** [review]

File: backoffice/config/settings/base.py (hunk 1 above)
Comment (DonHaul) [reply backoffice/config/settings/base.py:335]: Dont think so. On the documentation all options are strings https://docs.allauth.org/en/dev/account/configuration.html#:~:text=ACCOUNT_EMAIL_VERIFICATION

Comment (drjova) [reply backoffice/config/settings/base.py:335]: 👍

**Pull request merged**